e2e/angular-core/src/ng-add.test.ts

```diff
@@ -129,12 +129,11 @@ describe('convert Angular CLI workspace to an Nx workspace', () => {
     const updatedPackageJson = readJson('package.json');
     expect(updatedPackageJson.description).toEqual('some description');
     expect(updatedPackageJson.scripts).toEqual({
-      ng: 'nx',
```

Comment:
Should we instead change this to something like `'nx': 'nx'` or simply remove it?

Reply:
Yeah I'm not too certain!

Reply:
This is from what `@angular/cli` generates?

I think we should leave it alone.